docs/deployment/README.md

```diff
@@ -170,60 +124,38 @@ These buckets do not need any non-default permissions to function with Cumulus,
 --------------
 ## Configure and Deploy the IAM stack
 
-##### Configure deployment with `<daac>-deploy/iam/config.yml`
+### Configure deployment with `<daac>-deploy/iam/config.yml`
 
-The `iam` configuration creates 6 [roles](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) and an [instance profile](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html) used internally by the Cumulus stack.
-
-The various config fields are described below with a sample `config.yml` at the end.
+The `iam` configuration creates 7 [roles](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) and an [instance profile](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html) used internally by the Cumulus stack.
 
 ------
 
-###### iam-deployment-name:
-
-The name (e.g. dev) of the the 'deployment' - this key tells kes which configuration set (in addition to the default values) to use when creating the cloud formation template[^4]
-
-###### prefix:
-
-This value will prefix CloudFormation-created IAM resources and permissions.
-
-**The cumulus stack name must start with `<prefix>`** [^5]
-
-###### stackName:
-
-The name of this iam stack in CloudFormation (e.g. <prefix>-iam).
-
-###### buckets:
-
-The buckets created in the [Create S3 Buckets](#create-s3-buckets) step. Buckets are defined in the config.yml with a key, name, and type. Types should be one of: internal, public, private, or protected. Multiple buckets of each type can be configured. A key is used for the buckets to allow for swapping out the bucket names easily.
-
-###### useNgapPermissionBoundary:
-
-If deploying to a NASA NGAP account, set `useNgapPermissionBoundary: true`.
+**Sample new deployment added to config.yml**:
 
-------
+Descriptions of the fields can be found in [IAM Configuration Descriptions](deployment/config_descriptions.md#iam-configuration).
 
-**Sample new deployment added to config.yml**:
+```
+<iam-deployment-name>:    # e.g. dev (Note: Omit brackets, i.e. NOT <dev>)
```

Comment:
Should we include an actual example? I feel like people still might get tripped up in terms of how to use their prefix in the bucket names

Reply:
Done. I made it more into an actual example.